src/libraries/System.Reflection.Metadata/src/System/Reflection/PortableExecutable/PEHeaderBuilder.cs

```diff
@@ -105,7 +105,7 @@ public static PEHeaderBuilder CreateLibraryHeader()
             return new PEHeaderBuilder(imageCharacteristics: Characteristics.ExecutableImage | Characteristics.Dll);
         }
 
-        internal bool Is32Bit => Machine != Machine.Amd64 && Machine != Machine.IA64 && Machine != Machine.Arm64;
+        internal bool Is32Bit => Machine != Machine.Amd64 && Machine != Machine.IA64 && Machine != Machine.Arm64 && Machine != Machine.LoongArch64;
```

Comment:
This change is not really necessary. This list only needs to contain 64-bit architectures that are supported by Windows. For example, S390x is 64-bit too and it is not in this list.

Could you please revert this change?

Reply:
OK,thanks.
I will do it now.